tap_quickbooks/schemas/bills.json

```diff
@@ -76,9 +76,10 @@
             ]
           },
           "LineNum": {
```

Comment:
Do we need a format change for the Line Number?

Reply:
The API doc suggests that the data type is Decimal. However, it also mentioned that it is a positive integer. https://developer.intuit.com/app/developer/qbo/docs/api/accounting/most-commonly-used/bill

Reply:
Reverted back changes for LineNum.